gatsby_fp/src/components/fieldComponents/Education.js

```diff
@@ -15,20 +15,25 @@ export default function Education({ educationList }) {
               dangerouslySetInnerHTML={{
                 __html: ` 
                ${
-                 (element.deg && !(element.degother))
+                 (element.deg && !element.degother)
                    ? `${element.deg}`
                    : ``
                }
                ${
-                (element.degother && !(element.deg))
+                (element.degother && !element.deg)
                    ? `${element.degother}`
                    : ``
                }
                ${
-                (element.degother && element.deg)
+                (element.degother && element.deg && element.deg !== "Other")
                    ? `${element.deg}, ${element.degother} `
```

Comment:
Based on earlier conversation, we do not need a scenario where we show both `deg` and `degother` separated by a comma. It can simply be like "if degother exists, then show degother, elseif deg exists, then show deg"